### PR TITLE
My feature

### PR DESCRIPTION
**Pull request by Anna-Myzukina**

### CODE REVIEW

**Comment by jofer86** [comment]
#Review In Progress By Jorge F. (TSE)

**Comment by jofer86** [comment]
# Hello Team, I'm Jorge and I will be conducting the review for your project. :wave: :wave: 

## Let us get into topic and start checking out your code and comparing it with the specifications set by [The Odin Project](https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown).

**1.-** The first things that comes to mi mind when I see the [your site](https://htmlpreview.github.io/?https://github.com/Anna-Myzukina/Smashing-Magazine/blob/my-feature/index.html), compared to what it is asked from The Odin Project which is:   " **Now make the background colors of those divs different shades of gray which correspond to their visual weight on the Smashing homepage. If you squint your eyes and look from your solution to their page, does your eye get drawn to the same places?**" Is that you used a variety of different colors, that differs from the intention of the project, which in essence is to understand the visual weight of every element on the site, that is why is asked for you to build a site with `<div>` elements in different shades of gray, going from the darkest (which will represent the item with the highest visual hierarchy) to the lightest which will be the one with a lesser impact. **Please take a good look at the site and rank the visual impact of every element in order to color the backgrounds with different shades of gray, so that the project is understandable in its essence **.

**2.-**  Now moving onto this part in The Odin Project especifications. ** Copy the basic structure of the page using empty semantic HTML elements that are appropriately sized and positioned. ** This part involves the use of  appropriate use of tags, ideally our semantic HTML is made out of distinct areas such as `<nav>`, `<main>` and `<footer>`, which from the code I'm seing I realize some of that is missing, the navigation bar in its entirety should be within a `<nav>` tag, also, the main part of your site should be contained within a `<main>` tag. Your `<footer>` tag is appropriate therefore is ok. More on semantic HTML can be found [HERE](https://www.w3schools.com/html/html5_semantic_elements.asp)

**3.-** Now moving on to the Microverse requirements, which are the following.


# Minimum Requirements Checklist

To consider the project 'complete' it must satisfy the following points:

### Project Specification Shortcut

#### Navigation bar

:warning: It should use the `<nav>` HTML tag. **As previously discussed this is part of the semantic HTML part of your markup, please adjust accordingly.**
:warning: It should have seven separate containers (`<div>` is recommended). **I can see here that you did an unordered list, you don't have to worry about the images just have some place holder image and text, what matters most for the project is to set up the visual hierarchy of the site**

##### one-third-width containers (the actual width of the container is not important)

:warning:  Three articles with a width of `33%` **I can see that the divs which contain said articles are 33.33% in width, the semantic HTML part is missing that is setting up a `<section>` tag that contains `<article>` tags as children of said section, kindly make the semantically appropriate changes to match the requirements**
:heavy_check_mark:  Each article should have a title and paragraph with placeholder text. ** This part is fine but remember that we need this to be within an `<article>`tag **

##### half-width containers (the actual width of the container is not important)

:warning: Four articles with a width of `50% ** Same observation as above the sizes of the containers are fine but we need to make it semantically correct, so please re-tag your divs with the `<section>`tag, as well as the `<div>` with the class container should probably be a `<section>` tag.**
:heavy_check_mark: Each container should have an image in the top left corner with the title of an article under it

##### two-thirds-width containers (the actual width of the container is not important)

:heavy_check_mark:  Larger articles should be positioned on the left of the section with just a title and a small paragraph

##### full-width containers (the actual width of the container is not important)

:heavy_check_mark:  Single articles should cover the full width of the section

#### Footer

:heavy_check_mark:  The links in the footer should cover the central part of the footer, both horizontally and vertically

The main content should be made from separate containers, stacked on top of each other and in the following arrangement:

:warning:  All content should be wrapped with `<main>` HTML tags
:warning:  Every child of main should be a `<section>` HTML tag pair
:warning:  Every child of section should be an `<article>` HTML tag pair

#### Footer

:heavy_check_mark: It should have an unorganized list `<ul>`
:warning: The links in the footer should be list items `<li>` positioned using the `float` or `flex` display property **The items where positioned using the inline-block property, while they should have been positioned using floats or flex**

## It was a good effort, but the main goal of the project was to do some thinking about visual hierarchy and how it should be represented, in this case with the use of gray colors, take your time and think which elements on the original site pop to your attention first, second and so on, and color them accordingly from darker to lighter gray.

# Happy Coding.
# Cheers, Jorge F.

**Comment by mosaaleb** [comment]
Reviewing

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
 # Smashing-Magazine
-PROJECT: DESIGN TEARDOWN HTML and CSS
+<h1>PROJECT: DESIGN TEARDOWN HTML and CSS</h1>
+<ul>
+<li>
```
Comment (mosaaleb) [inline README.md:4]: While you definitely can write `html` inside markdown files, it's always better to use the [markdown syntax](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). Markdown syntax is really easy to learn, it's even easier than html.

**Review by mosaaleb — APPROVED** [review]
## Hi @Anna-Myzukina, @maya88en :wave:

### Amazing work team. All requirements are met perfectly. This project is done professionaly and with dedication :clap:

[Optional] See my comment below regarding markdown.

You don't have to send a new code review for this, but consider using markdown syntax in coming projects instead of `html`

You can merge into master :+1:

Good luck with the next projects.

### Happy coding :computer:

**Pull request merged**